tests/project_directory_test.php

```diff
@@ -107,4 +107,20 @@ public function testGetDirectoryAvoidPluginsDirectory()
         $this->assertInstanceOf('\\Dxw\\Whippet\\ProjectDirectory', $result->unwrap());
         $this->assertEquals($dir, $result->unwrap()->__toString());
     }
+
+    public function testGetDirectoryWithEnv()
+    {
+        $dir = $this->getDir();
+
+        putenv('WHIPPET_PROJECT_DIRECTORY=/etc');
```

Comment:
Ah cool. I'd seen the use of vfs in php tests before, but only explicitly. It's cool that you can do this transparently.